modules/data.atmosphere/R/download.GFDL.R

```diff
@@ -14,7 +14,7 @@
 #' @param scenario Which scenario to run (options are rcp26, rcp45, rcp60, rcp85)
 #' @param ensemble_member Which ensemble_member to initialize the run (options are r1i1p1, r3i1p1, r5i1p1)
 #' @author James Simkins, Alexey Shiklomanov, Ankur Desai
-download.GFDL <- function(outfolder, start_date, end_date, site_id, lat.in, lon.in,
+download.GFDL <- function(outfolder, start_date, end_date, lat.in, lon.in,
```

Comment:
Are you sure removing this is safe with met.process?

Reply:
@ankurdesai Does met.process have any expectations about how download.* will use the argument? I'm sure it's safe to remove in the sense that it won't throw an error (a `site_id` passed from upstream will be accepted, treated as part of `...`, and ignored) or change the behavior of  `download.GFDL` (it was already ignoring `site_id`), so removing it is only a problem if ignoring it is also a problem.

Reply:
Just checked a few other met download functions and `site_id` doesn't occur in most of them, and in the few that contain it may be in error (GLDAS, NLDAS) as they're using `site_id` to append to the outfolder name, but I suspect (hope) that's already part of the outfolder name that's being passed in.